Bonsai.Configuration/ScriptExtensions.cs

```diff
@@ -98,6 +100,12 @@ static XmlReaderSettings GetXmlReaderSettings()
             };
         }
 
+        static XElement GetProperty(XDocument document, string key)
+            => document.XPathSelectElement($"/Project/PropertyGroup/{key}");
+
+        static bool? GetBoolProperty(XDocument document, string key)
+            => GetProperty(document, key)?.Value?.Equals("true", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
The logic here was to make the caller consider what the default should actually be when the property is absent since it might not be `false`, but it almost always is so that's fine

Reply:
(Also after letting that thought bounce around in my head, that logic doesn't actually reflect how MSBuild works regardless since it doesn't actually have booleans.)

Reply:
Done in refactored version